lib/core/utils/parse-crossorigin-stylesheet.js

```diff
@@ -19,6 +19,7 @@ axe.utils.parseCrossOriginStylesheet = function parseCrossOriginStylesheet(
 ) {
 	const axiosOptions = {
 		method: 'get',
+		timeout: 5000,
```

Comment:
Rather than an arbitrary number, we should use https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/preload.js#L32

Because if I want my preload to kick the bucket in `3000` instead & it does not then that is not ideal